### PR TITLE
[SITES-364] Feature/view news from slash news

### PR DESCRIPTION
**Pull request by ltankey**
Provides an aggregation of all news items on `/news`.

image

### CODE REVIEW

**Comment by ltankey** [comment]
A rebase from another branch has created more commits than were actually done here. This shouldn't be a problem when we squash.

File: app/models/news_article.rb [hunk 1]
```diff
@@ -8,6 +8,16 @@ class NewsArticle < Node
...
 
+  scope :by_release_date, -> {
+    order("data ->> 'release_date' DESC")
+  }
+
+  scope :by_name, -> {
+    order("content ->> 'name' ASC")
+  }
+
+  scope :published, -> { where state: 'published' }
```
Comment (micapam) [inline app/models/news_article.rb]: Isn't this already in Node?

Comment (ltankey) [inline app/models/news_article.rb]: Good pick-up - it wasn't originally. Removing now.

**Comment by micapam** [comment]
One trivial comment aside, this LGTM

File: config/breadcrumbs.rb [hunk 2]
```diff
@@ -45,6 +45,16 @@
...
+crumb :public_news_articles do |section|
+  link 'News', news_articles_path(section.slug)
+  parent :public_node, section.home_node
+end
+
+crumb :public_news_article do |node|
+  link node.name, news_article_path(node.section.slug, node.release_date, node.slug)
+  parent :public_news_articles, node.section
+end
+
```
Comment (tessereth) [inline config/breadcrumbs.rb]: These are duplicated at the end of the file

File: spec/models/node_spec.rb [hunk 3]
```diff
@@ -184,4 +184,21 @@
...
+        expect(article.full_path).to eq(Rails.application.routes.url_helpers.news_article_path(article.section.home_node.slug, article.slug))
+      end
+    end
+
+    context 'for a general node' do
+      it 'matches its url helper' do
+        expect(node.full_path).to eq(Rails.application.routes.url_helpers.nodes_path(node.path))
+      end
+    end
+  end
```
Comment (tessereth) [inline spec/models/node_spec.rb]: this is also duplicated

**Comment by maxious** [comment]
Changelog?

**Comment by ltankey** [comment]
Thanks @maxious, added.

**Comment by maxious** [comment]
LGTM

**Pull request merged**